src/component/Chat/Chat.js

```diff
@@ -1,57 +1,72 @@
 import React, { useEffect, useState } from "react";
 import socketIo from "socket.io-client";
-import Sidebar from "./SideBar/SideBar";
-import { user } from "../userJoin/Join.js";
 import TopBar from "./TopBar/TopBar";
 import ChatArea from "./ChatArea/ChatArea";
 import "./Chat.css"; // Import the CSS file
 import axios from "axios";
 
 let socket;
 const ENDPOINT = process.env.REACT_APP_ENDPOINT;
```

Comment:
@athar-shayan instead of fetching directly from env file, create constant file. look into ems project

Reply:
Okay sir I will look into it.